prelude/README.md

```diff
@@ -111,6 +122,16 @@ instance categoryArr :: Category Prim.Function
 ($) :: forall a b. (a -> b) -> a -> b
 ```
 
+Applies a function to its argument
+
+```purescript
+    (+) 1 ((*) 2 (const 1 2)) = (+) 1 $ (*) 2 $ const 1 2
```

Comment:
I think it's fine to take examples from the core libraries, or just to make up functions from a standard domain like product/customer/category or something.